packages/arcgis-rest-developer-credentials/test/getApiKey.test.ts

```diff
@@ -0,0 +1,358 @@
+import fetchMock from "fetch-mock";
+import { ArcGISIdentityManager } from "@esri/arcgis-rest-request";
+import { IItem } from "@esri/arcgis-rest-portal";
+import { getApiKey, IRegisteredAppResponse, IApiKeyResponse } from "../src";
+import { TOMORROW } from "../../../scripts/test-helpers";
```

Comment:
This needs to be fixed but there are other Node test errors that I'll comment on.

```suggestion
import { TOMORROW } from "../../../scripts/test-helpers.js";
```